<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_setting_sample_time</class>
 <widget class="QDialog" name="view_setting_sample_time">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>248</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>248</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>248</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting Sample Time</string>
  </property>
  <widget class="QLabel" name="sample_time_step_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>90</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample time step:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_simulator_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample time simulator:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_controller_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>113</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample time controller:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_graph_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>97</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample time graph:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_simulator_step_label">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>70</y>
     <width>40</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Step  x</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_controller_step_label">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>120</y>
     <width>40</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Step  x</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_graph_step_label">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>170</y>
     <width>40</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Step  x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>70</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>120</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>170</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_step_unite_mesure">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>20</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_simulator_unite_mesure">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>70</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_controller_unite_mesure">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>120</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_graph_unite_mesure">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>170</y>
     <width>20</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_time_step_value">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>17</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_time_simulator_step_value">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>67</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_time_controller_step_value">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>117</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="sample_time_graph_step_value">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>167</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_simulator_value">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>38</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_controller_value">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>38</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_time_graph_value">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>38</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>208</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>208</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
